rasa_nlu/tokenizers/jieba_tokenizer.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Text
+
+from rasa_nlu.config import RasaNLUConfig
+from rasa_nlu.tokenizers import Tokenizer, Token
+from rasa_nlu.components import Component
+from rasa_nlu.training_data import Message
+from rasa_nlu.training_data import TrainingData
+
+
+class JiebaTokenizer(Tokenizer, Component):
+    name = "tokenizer_jieba"
+
+    provides = ["tokens"]
+
+    def train(self, training_data, config, **kwargs):
+        # type: (TrainingData, RasaNLUConfig, **Any) -> None
+
```

Comment:
I added the following in the training function in jieba_tokenizer:
``
if config['language'] != 'zh':
    raise Exception("tokenizer_jieba is only used for Chinese. Check your configure json file.")
```

However now some pytests now fail since it will run through tokenizer_jieba with language set as default "en" during "all_components" pipeline.
May I ask what is the best way to work around this? Should I also add language check in those pytests?